config/externalname.go

```diff
@@ -437,17 +437,22 @@ var TerraformPluginSDKExternalNameConfigs = map[string]config.ExternalName{
 
 	// eks
 	//
-	"aws_eks_cluster": config.NameAsIdentifier,
-	// Imported using the cluster_name and node_group_name separated by a
-	// colon (:): my_cluster:my_node_group
-	"aws_eks_node_group": config.TemplatedStringAsIdentifier("node_group_name", "{{ .parameters.cluster_name }}:{{ .external_name }}"),
 	// my_cluster:my_eks_addon
 	// "aws_eks_addon": config.TemplatedStringAsIdentifier("addon_name", "{{ .parameters.cluster_name }}:{{ .external_name }}"),
 	"aws_eks_addon": FormattedIdentifierFromProvider(":", "cluster_name", "addon_name"),
+	// import EKS access entry using the cluster_name and principal_arn separated by a colon (:).
+	"aws_eks_access_entry": FormattedIdentifierFromProviderWithIdentifierFields(":", "cluster_name", "principal_arn"),
```

Comment:
As I mentioned [here](https://github.com/crossplane-contrib/provider-upjet-aws/pull/1273#discussion_r1653641930), could you please try the following:

```suggestion
	"aws_eks_access_entry": config.TemplatedStringAsIdentifier("", "{{ .parameters.cluster_name }}:{{ .parameters.principal_arn }}"),
```